script.js

```diff
@@ -61,11 +61,29 @@ function checkEmail(Diana) {
   }
   return false;
 }
+const custmr = document.getElementById('name');
+const txt = document.getElementById('message');
+
+function storeData() {
+  const testObject = {
+    userID: custmr.value,
+    email: email.value,
+    message: txt.value,
+  };
+  localStorage.setItem('store', JSON.stringify(testObject));
+}
+const emptyObject = JSON.parse(localStorage.getItem('store'));
+
+custmr.value = emptyObject.userID;
+email.value = emptyObject.email;
+txt.value = emptyObject.message;
```

Comment:
You have done greatly at attempting to load the data from local storage when the page loads. However, for a user who has never saved before, performing localStorage.getItem will return null, hence the error below:

![image](https://user-images.githubusercontent.com/20554583/190863991-db957ee1-e9ea-421e-a4bd-a6fa37f327de.png)

A way around this could be initializing the object the store returns null.

e.g
```
storeInit  = {
   userID: '',
   email: '',
   message: ''
}

const emptyObject = JSON.parse(localStorage.getItem('store')) || storeInit
```